cx_Freeze/finder.py

```diff
@@ -218,7 +218,7 @@ def _FindModule(self, name, path, namespace):
                 # Namespace package (?)
                 module = sys.modules[name]
                 info = ("", "", imp.PKG_DIRECTORY)
-                return None, module.__path__[0], info
+                return None, next(p for p in module.__path__), info
```

Comment:
I am wondering if anyone was able to reproduce this issue.  I am currently experiencing the same namespace package issue.  None of these fixes seem to correct the issue.  I am trying to import custom namespaces that I have written, and it definitely isn't finding them either.  I am running on RHEL 7.3, Python 3.4.2, and cx_Freeze 5.0.1.

My __init__.py files are using the standard pre-PEP420 import for setuptools:

__import__('pkg_resources').declare_namespace(__name__)

The __init__.py files have been removed from the site-packages directories, and they play nicely with pip.